scipy/stats/_stats_py.py

```diff
@@ -6762,8 +6762,9 @@ def ttest_ind(a, b, *, axis=0, equal_var=True, nan_policy='propagate',
         raise NotImplementedError(message)
 
     result_shape = _broadcast_array_shapes_remove_axis((a, b), axis=axis)
-    NaN = xp.full(result_shape, _get_nan(a, b, xp=xp))
```

Comment:
fill_value can't be an Array (discussion [here](https://github.com/data-apis/array-api/issues/909) to allow it).
This triggers a warning in Dask, which however doesn't show up until https://github.com/data-apis/array-api-extra/pull/284 will be merged later, because the Dask graph is never computed.